Model:
  classname: neurorobotics_dl.models.EEGNet
  path: /home/curtaz/Neurorobotics/models/antneuro32ch_generic
  options:
    nb_classes: 32
    Chans: 32
    Samples: 512 # typically set to fs
    dropoutRate: 0.5
    kernLength: 256 # typically set to fs/2
    F1: 8
    D: 2
    F2: 16
  
Antneuro16Generic:
  classname: neurorobotics_dl.models.EEGNet
  path: /home/curtaz/Neurorobotics/models/antneuro16ch_generic
  options:
    nb_classes: 2
    Chans: 16
    Samples: 512 # typically set to fs
    dropoutRate: 0.5
    kernLength: 256 # typically set to fs/2
    F1: 8
    D: 2
    F2: 16

Antneuro32Generic:
  classname: neurorobotics_dl.models.EEGNet
  path: /home/curtaz/Neurorobotics/models/antneuro32ch_generic
  options:
    nb_classes: 32
    Chans: 32
    Samples: 512 # typically set to fs
    dropoutRate: 0.5
    kernLength: 256 # typically set to fs/2
    F1: 8
    D: 2
    F2: 16

